snippets/HSVToRGB.md

```diff
@@ -0,0 +1,20 @@
+---
+title: HSVToRGB
+tags: math, intermediate
+---
+
+Converts the values of HSV color components to RGB components.
+
+Convert given HSV parameters into RGB ones and returns it in array format. 
```

Comment:
The new format suggests to use bullet points. I think you should:

- Explain the ranges of the values in one bullet point.
- Link to a wikipedia page of the calculation and/or format as reference.